### PR TITLE
[Solved #105] Race end controllers

### PR DESCRIPTION
**Pull request by supunTE**

### CODE REVIEW

File: apps/client/src/pages/race/Race.page.tsx [hunk 1]
```diff
@@ -107,6 +110,13 @@ export function Race(): ReactElement {
     }
   }, [raceReadyTime, raceId]);
 
+  const raceTimeEndHandler = (): void => {
```
Comment (dalanad) [inline apps/client/src/pages/race/Race.page.tsx:113]: `raceEndHandler`

File: libs/models/src/lib/sockets/protocols.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ enum ProtocolName {
   SendTypeLog = 'SEND_TYPE_LOG',
   UpdateTypeLogs = 'UPDATE_TYPE_LOGS',
   Timeout = 'TIMEOUT',
-  ForceEnd = 'FORCE_END',
+  // ForceEnd = 'FORCE_END',
```
Comment (dalanad) [inline libs/models/src/lib/sockets/protocols.ts:28]: Why is this ?

**Review by dalanad — CHANGES_REQUESTED** [review]

File: libs/models/src/lib/sockets/protocols.ts (hunk 2 above)
Comment (supunTE) [reply libs/models/src/lib/sockets/protocols.ts:28]: This was initially intended to be a separate protocol. However, I later discovered we could handle this event within sendLeaderboard. I kept the protocol until the implementation of sendLeaderboard was completed. It has now been removed in the latest PRs.
ab2b857b81490c2cee3f0476ca2cd3beae5cc2aa

File: apps/client/src/pages/race/Race.page.tsx (hunk 1 above)
Comment (supunTE) [reply apps/client/src/pages/race/Race.page.tsx:113]: renamed in #114 5feab21e8a824365acd9e9de9cfe2cd16090c19e

**Review by dalanad — APPROVED** [review]

**Pull request merged**